maestro-agent/src/main/groovy/org/maestro/agent/ext/requests/get/Get.groovy

```diff
@@ -0,0 +1,14 @@
+package org.maestro.agent.ext.requests.get
+
+import org.maestro.agent.base.AbstractHandler
+import org.maestro.client.exchange.MaestroTopics
+import org.maestro.client.notes.OkResponse
+
+class GetHandler extends AbstractHandler {
+
+    @Override
+    Object handle() {
+        this.getClient().publish(MaestroTopics.MAESTRO_TOPIC, new OkResponse())
```

Comment:
I don't think we need to be returning a response all the time. I think a proper response must only be sent if the command was actually handled by the agent. Therefore, the method code should only contain return null. The exception for this are: halt and ping which must  be handled.